lib/glimesh_web/live/chat_live/index.ex

```diff
@@ -39,8 +39,9 @@ defmodule GlimeshWeb.ChatLive.Index do
       |> assign(:permissions, Chat.get_moderator_permissions(channel, session["user"]))
       |> assign(:chat_messages, list_chat_messages(channel))
       |> assign(:chat_message, %ChatMessage{})
+      |> assign(:show_timestamps?, (if session["user"], do: session["user"].show_timestamps?, else: false))
 
-    {:ok, new_socket, temporary_assigns: [chat_messages: []]}
```

Comment:
I think this had something to do with timing out messages, can you try with two accounts and then time one out?

Reply:
Timed out 2nd account and the message deleted for both views, not sure what it did previously without it.